### PR TITLE
Add fix script for descriptions

### PR DESCRIPTION
**Pull request by queengooborg**
This PR adds a fix script for the descriptions, so that description issues can be automatically fixed.

### CODE REVIEW

File: lint/fixer/descriptions.ts [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+ * See LICENSE file for more information. */
+
+import fs from 'node:fs';
+
+import { IS_WINDOWS } from '../utils.js';
+import testDescriptions, { processData } from '../linter/test-descriptions.js';
+import walk from '../../utils/walk.js';
+
+/**
+ * Removes irrelevant flags from the compatibility data of a specified file
```
Comment (Elchi3) [inline lint/fixer/descriptions.ts:11]:
```suggestion
 * Fixes issues with descriptions in API data
```

**Review by Elchi3 — CHANGES_REQUESTED** [review]
Thanks for working on this!

There is a problem with the description test:
In `lint/linter/test-descriptions.ts` line 119 you need to have something like ` if (data.description && typeof data.description === 'string') {` so that it doesn't get hung up on data with the key "description". See `api.GPUAdapterInfo.description`.

Also jsdoc needs to be corrected, see suggestion.

**Comment by queengooborg** [comment]
> There is a problem with the description test:
> In `lint/linter/test-descriptions.ts` line 119 you need to have something like ` if (data.description && typeof data.description === 'string') {` so that it doesn't get hung up on data with the key "description". See `api.GPUAdapterInfo.description`.

The `data` variable is always going to be the `__compat` object, so we don't have to worry about checks like this!  https://github.com/mdn/browser-compat-data/blob/3fa1f7630e9d7492f6cb619a69da1dba1cca593e/lint/lint.ts#L147-L158

**Comment by Elchi3** [comment]
Ah okay, very good. I'm getting an error running `npm run fix` on this branch, though, and it seems to hiccup when it sees `api.GPUAdapterInfo.description`

```
> @mdn/browser-compat-data@5.5.31 lint:fix
> node --loader=ts-node/esm --no-warnings=ExperimentalWarning lint/fix.ts

file:///Users/florianscholz/dev/browser-compat-data/lint/linter/test-notes.ts:50
    if (string.includes('  ')) {
               ^

TypeError: string.includes is not a function
    at validateHTML (file:///Users/florianscholz/dev/browser-compat-data/lint/linter/test-notes.ts:50:16)
    at processData (file:///Users/florianscholz/dev/browser-compat-data/lint/linter/test-descriptions.ts:59:24)
    at fixDescriptions (file:///Users/florianscholz/dev/browser-compat-data/lint/fixer/descriptions.ts:17:24)
    at load (file:///Users/florianscholz/dev/browser-compat-data/lint/fix.ts:39:21)
    at async load (file:///Users/florianscholz/dev/browser-compat-data/lint/fix.ts:48:13)
    at async file:///Users/florianscholz/dev/browser-compat-data/lint/fix.ts:56:9

Node.js v20.13.1
```

**Comment by queengooborg** [comment]
Ah, looks like there was a variable issue, fixed!

**Comment by Elchi3** [comment]
Fantastic, that looks better!

Now I'm getting a warning, though:
> File /Users/florianscholz/dev/browser-compat-data/test doesn't exist!

**Comment by queengooborg** [comment]
That warning was from another bug that has been fixed upstream!

**Review by Elchi3 — APPROVED** [review]
Great, then this is good to merge! Thank you! 👍

**Pull request merged**